src/libraries/System.Runtime.Serialization.Schema/ref/System.Runtime.Serialization.Schema.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="System.Runtime.Serialization.Schema.cs" />
+  </ItemGroup>
+
+  <!-- NOTE TODO smolloy - Is this the right way to include CodeDom? Probably not. Feels like it should be a nuget dependency somehow.
+    But we want to bring in the package from this repo, not just any package.
+    Let's just do it this way for now and revisit once we've got all the rest of the heavy lifting done.
```

Comment:
This is the correct way.  It becomes a nuget dependency when `dotnet pack` sees that the dependent project sets `IsPackable`.